lib/local_system.rb

```diff
@@ -124,4 +124,26 @@ def read_file(file)
     # File not found, return nil
     return
   end
+
+  # Copies a file to the local system
+  def inject_file(source, destination_dir)
+    begin
+      FileUtils.copy(source, destination_dir)
+    rescue
+      raise Machinery::Errors::InjectFileFailed.new(
+        "Could not inject file '#{source}' to local system.\n" \
+        "Error: #{e}"
+      )
+    end
+  end
+
+  # Removes a file from the System
+  def remove_file(file)
+    File.delete(file)
+  rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?